docs/resources/Channel.md

```diff
@@ -640,6 +640,11 @@ Embed types are "loosely defined" and, for the most part, are not used by our cl
 | proxy_url     | string    | a proxied url of file                                                               |
 | height?       | ?integer  | height of file (if image)                                                           |
 | width?        | ?integer  | width of file (if image)                                                            |
+| ephemeral?    | boolean   | whether this attachment is ephemeral                                                |
+
+###### Attachment Reference
+
+Ephemeral attachments will automatically be removed after a set period of time. Ephemeral attachments on messages are guaranteed to be available as long as the message itself exists.
```

Comment:
Wouldn't it make more sense to use a footnote?

```suggestion
| ephemeral? \* | boolean   | whether this attachment is ephemeral                                                |

\* Ephemeral attachments will automatically be removed after a set period of time. Ephemeral attachments on messages are guaranteed to be available as long as the message itself exists.
```

Reply:
I guess too